bridges/bin/millau/runtime/src/lib.rs

```diff
@@ -279,10 +279,19 @@ parameter_types! {
 	pub LeafVersion: MmrLeafVersion = MmrLeafVersion::new(0, 0);
 }
 
+pub struct ParachainHeadsProvider;
+
+impl BeefyDataProvider for ParachainHeadsProvider {
+	fn extra_data() -> Vec<u8> {
+		Vec::new()
+	}
+}
+
+
 impl pallet_beefy_mmr::Config for Runtime {
 	type LeafVersion = LeafVersion;
 	type BeefyAuthorityToMerkleLeaf = pallet_beefy_mmr::BeefyEcdsaToEthereum;
-	type ParachainHeads = ();
+	type BeefyDataProvider = ParachainHeadsProvider;
```

Comment:
actually provide a `BeefyDataProvider` implementation for `()` in the substrate repo